### ISSUE TITLE
Improve the code

### ISSUE THREAD

**Issue by jamesli2021** [corbado/example-passkeys-astro#5]
To improve the speed of development and production, we don't need `type` and `define:vars` parameters and simplify the code to just:

```
<script>
const projectId = import.meta.env.DEV ? import.meta.env.PUBLIC_PROJECT_ID_DEV : import.meta.env.PUBLIC_PROJECT_ID_PROD;

...
</script>
```

**Comment by vincentdelitz** [comment]
@AmalADDAD98 can you check this one please

**Comment by AmalADDAD98** [comment]
@jamesli2021 Thank you for the refactoring suggestion. I reviewed and implemented it, but encountered an issue: the environment variable returned an "Undefined" error during client-side execution.

**Explanation:**

In Astro, environment variables are typically accessed using `import.meta.env`, which works well in server-side contexts. However, when these variables need to be accessed in a client-side script, they must be explicitly passed. And, Astro provides the `define:vars` directive, which is useful for passing server-side variables to client-side scripts. (https://docs.astro.build/en/reference/directives-reference/#definevars). This is the approach recommended by Astro if we want to keep the environment variables managed by Astro’s environment handling.

**Comment by jamesli2021** [comment]
Hmm, strange, I got it working. It will be a error when you want to `import Corbado from "@corbado/web-js";`.

My solution is using:
```
<script>
import Corbado from "@corbado/web-js";

...{
projectId: import.meta.env.DEV ? import.meta.env.PUBLIC_PROJECT_ID_DEV : import.meta.env.PUBLIC_PROJECT_ID_PROD,
}...

</script>
```

**Comment by jamesli2021** [comment]
My assumption that getting an `undefined` value could be due to `.env.example`, if the right file name should be `.env`.

**Comment by AmalADDAD98** [comment]
> Hmm, strange, I got it working. It will be a error when you want to `import Corbado from "@corbado/web-js";`.
> 
> My solution is using:
> 
> ```
> <script>
> import Corbado from "@corbado/web-js";
> 
> ...{
> projectId: import.meta.env.DEV ? import.meta.env.PUBLIC_PROJECT_ID_DEV : import.meta.env.PUBLIC_PROJECT_ID_PROD,
> }...
> 
> </script>
> ```

@jamesli2021 Thank you for sharing your solution. It would be beneficial if could create a PR so I can review your suggestion. As I explained, the best practice is to keep the environment variables handled by the Astro environment, and to pass them to client-side scripts via the `define:vars` directive.

**Comment by AmalADDAD98** [comment]
> My assumption that getting an `undefined` value could be due to `.env.example`, if the right file name should be `.env`.

@jamesli2021 No, I do have a correctly named `.env` file locally, containing the appropriate variables. The `.env.example` file is a template to illustrate the required environment variables.
